### ISSUE TITLE
FTBFS with gpsd 3.18.1

### ISSUE THREAD

**Issue by CameronNemo** [navit-gps/navit#711]
```
/builddir/navit-0.5.3/navit/vehicle/gpsd/vehicle_gpsd.c: In function 'vehicle_gpsd_io':
/builddir/navit-0.5.3/navit/vehicle/gpsd/vehicle_gpsd.c:328:28: error: too few arguments to function 'gps_read'
         while((read_result=gps_read(priv->gps))>0);
                            ^~~~~~~~
In file included from /builddir/navit-0.5.3/navit/vehicle/gpsd/vehicle_gpsd.c:21:
/usr/include/gps.h:2122:12: note: declared here
 extern int gps_read(struct gps_data_t *, char *message, int message_len);
            ^~~~~~~~
make[2]: *** [navit/vehicle/gpsd/CMakeFiles/vehicle_gpsd.dir/build.make:63: navit/vehicle/gpsd/CMakeFiles/vehicle_gpsd.dir/vehicle_gpsd.c.o] Error 1
```

**Comment by CameronNemo** [comment]
Similar to #649, which somebody claims to have a patch for.

**Comment by jkoan** [comment]
Should be fixed with #735
